CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### citus-docker v9.1.1.docker (December 19, 2019) ###
+
+* Bump Citus version to 9.1.1
+
+* Bump PostgreSQL version to 
```

Comment:
This line shouldn't come up. Did you try to update the PG version by supplying the optional param?

Is there something that needs to be fixed in the automation scripts?

Reply:
I run update script by providing PG version by supplying the optional param, there is no problem in scripts